models/users.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../db/connection');
+
+async function selectUsers() {
+    try {
+        const users = await db.query(`
+        SELECT * FROM users;`)
+        return users.rows;
+    } catch (error) {
+        
```

Comment:
No catch needed in model, you can just remove the try/catch block and have lines 5-7 instead. Any errors happening from the model will be caught by the catch block in the controller because the call of our model is located in the try block (but see above - you need to call next with the error for it to connect back to your app and error middleware. Really important!)